frontend/src/routes/+layout.svelte

```diff
@@ -17,17 +17,17 @@
 	function closeOnEscape(event: KeyboardEvent) {
 		event.key === 'Escape' && close()
 	}
-
+	
 	// https://www.w3.org/TR/trace-context/#traceparent-header
 	// so the page-load instrumentation can be correlated with the server load
-	const traceParent = get_trace_parent()
+	export let data: LayoutData;
```

Comment:
I'm not sure the comment here is relevant to data here. Maybe in the load function?